Project_source/app/src/main/java/android/clicker/school_live_simulator/BicycleState.kt

```diff
@@ -0,0 +1,34 @@
+package android.clicker.school_live_simulator
+
+/**
+ * @author      Andrey Novikov andrewnovicov@gmail.com
+ * @version     1.0                 (current version number of program)
+ * @since       1.0          (the version of the package this class was first added to)
+ */
```

Comment:
Remove this field please and don't write commentaries with author, version and since

Reply:
Fixed